src/PerformanceTests/Common/Program.cs

```diff
@@ -30,7 +30,7 @@ static int Main()
         {
             ServicePointManager.Expect100Continue = false;
             ServicePointManager.UseNagleAlgorithm = false;
-            ServicePointManager.DefaultConnectionLimit = 150;
+            ServicePointManager.DefaultConnectionLimit = Settings.DefaultConnectionLimit;
```

Comment:
In this case a code comment on why this is done would help to prevent someone refactoring this in the future.

Reply:
Feel free to add a comment if you think it is important.